node/database-cleaner/src/utils.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Calculate and return the expiry date based on the retention period.
+ * @returns {Date} The calculated expiry date.
+ */
+export function getExpiryDate() {
+  const retentionPeriod = Number(process.env.RETENTION_PERIOD_DAYS ?? 30);
+  const expiryDate = new Date();
+  expiryDate.setDate(expiryDate.getDate() - retentionPeriod);
+  return expiryDate;

```

Comment:
You can just do `return new Date(Date.now() - retentionPeriod * 24 * 60 * 60 * 1000)` here

Reply:
Implemented :)